src/Service/AlertManager.php

```diff
@@ -123,8 +123,10 @@ public function getServiceAlerts(EntityInterface $node) {
     // Get alerts without location assigned.
     $query = $this->nodeStorage->getQuery()
       ->condition('type', 'alert')
-      ->condition('status', 1)
-      ->notExists('field_alert_location');
+      ->condition('status', 1);
+    if (\Drupal::database()->schema()->tableExists('node__field_alert_location')) {
```

Comment:
Let's pull database service via Dependency Injection, not via static call.

Reply:
See @froboy

Reply:
@podarok resolved.